apps/storybook/src/Breadcrumbs.stories.tsx

```diff
@@ -88,6 +101,93 @@ export const Overflow: Story<BreadcrumbsProps> = (args) => {
   );
 };
 
+export const CustomOverflowBackButton: Story<BreadcrumbsProps> = (args) => {
```

Comment:
I am not sure about this story since it is not very clear what it does because you need to go to Actions tab. I would say better remove it.

Reply:
I added a tooltip to make it more clear. I would like to keep this example as this use case was specifically requested.